src/config.js

```diff
@@ -61,7 +71,18 @@ class Config {
       fetchMaxAttempts !== undefined && fetchMaxAttempts !== null
         ? fetchMaxAttempts
         : 4
+    this.throttleConfig = {
+      throttleRequests: throttleRequests || false,
+      throttleLimit: throttleLimit || 3,
+      throttleInterval: throttleInterval || 125,
+      throttleStrict: throttleStrict || false,
```

Comment:
What's the purpose of these values, is it so the user of the sdk can check what the throttle was set to when it was created?

Reply:
Yes, but if it's not necessary we can remove it, thank you